src/PublicApiGenerator/CodeNormalizer.cs

```diff
@@ -51,6 +51,7 @@ public static string NormalizeGeneratedCode(StringWriter writer)
 
             gennedClass = gennedClass.Replace("class " + StaticMarker, "static class ");
```

Comment:
Should we also make sure to remove the static marker?

Reply:
and maybe while we are at it also remove `AttributeMarker `

Reply:
static ctor is not a part of public api, I added tests

Reply:
> and maybe while we are at it also remove AttributeMarker

Let's remove it only if/when we see such a situation.